staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt-config.go

```diff
@@ -201,7 +200,16 @@ func (reqMgmt *requestManagementSystem) digestConfigObjects(newPLs []*rmtypesv1a
 			klog.V(5).Infof("Using exempt priority level %s: quiescent=%v", plName, plState.emptyHandler != nil)
 			continue
 		}
-		plState.concurrencyLimit = int(math.Ceil(float64(reqMgmt.serverConcurrencyLimit) * float64(plState.config.AssuredConcurrencyShares) / shareSum))
+		// TODO(aaron-prindle) HACK/DEBUG/CHANGE - this is always -9223372036854775808
+		// in my tests.  I imagine I am not bootstrapping everything properly
+		// I have currently hacked/modified this to be the regMgmt.serverConcurrencyLimit value
+
+		// these values are always 0 w/ my incorrect bootstrapping
```

Comment:
This problem should not still be with us.  But I know there are bugs that got merged into the feature branch.  To help debug, how about
```
klog.V(7).Infof("shares=%v, shareSum=%v", plState.config.AssuredConcurrencyShares, shareSum)
```